### DISCUSSION TITLE
Allow for positive and negative filters at the same time

### DISCUSSION THREAD

**Discussion by naxey** [roovo/obsidian-card-board#174] [Ideas]
The current settings only allow to use of an `Allow` **or** a `Deny` list.

Providing two separate text boxes for definitions would allow for more granular control. Especially would be especially useful for big folders with only one subfolder that should not show up.

For reference, this is the settings currently implemented:
<img width="784" alt="image" src="https://github.com/roovo/obsidian-card-board/assets/40920984/59936ad5-9ff9-49f7-b336-9adbfe2dd865">

**Comment by roovo** [comment]
Yes!!  Totally see this, it is something I have thought about myself too -> it would be great if there was some way of building these up with a list of things combined with some basic boolean operators (and, or, not).

I will create an issue from this discussion as I don't think I have one for this and it will keep in on my radar better!

Thank you

**Reply by roovo** [reply]
[issue #199 added]
